plugins/session_notifier.rb

```diff
@@ -205,7 +237,8 @@ def load_settings_from_config
           @minimum_ip       = IPAddr.new(group['minimum_ip']) if group['minimum_ip']
           @maximum_ip       = IPAddr.new(group['maximum_ip']) if group['maximum_ip']
           @dingtalk_webhook = group['dingtalk_webhook']       if group['dingtalk_webhook']
-
+          @gotify_address   = group['gotify_address']         if group['gotify_address']
+          @gotify_ssl       = group['gotify_ssl']             if group['gotify_ssl']
```

Comment:
Instead of having the address and SSL as different options, can you use a base URL like `https://mygotify.com/`. This would also allow the user to specify a port too all in the same option.

Reply:
Thanks for your advice~